library/Reporting/Web/Forms/ScheduleForm.php

```diff
@@ -135,6 +141,14 @@ protected function assemble()
             'label' => $this->id === null ? $this->translate('Create Schedule') : $this->translate('Update Schedule')
         ]);
 
+        $action = new SendMail();
+        $sendButton = $this->createElement('submit', 'send', [
+            'label'          => $action->getName(),
```

Comment:
You should use `$this->translate(Send Report Now)` like the PR title and not the action name.